test/cases/jsx/jsx.spec.js

```diff
@@ -53,7 +53,7 @@ describe('Run WCC For ', function() {
 
           expect(badge.getAttribute('count')).to.be.equal('0');
           expect(span.getAttribute('class')).to.be.equal('unmet');
-          expect(span.textContent).to.be.equal('0');
+          expect(span.textContent.trim()).to.be.equal('0');
```

Comment:
I could have done a replace on whitespace, but I figured this was simpler for now.